community/assessments/projects/kubescape/self-assessment.md

```diff
@@ -0,0 +1,337 @@
+* Synchornizer server: Synchornizer server can trigger the *Synchronizer* component to store objects in Kubernetes API (this is limited by the *ClusterRole* the *Synchronizer* is assigned with)
+
+### Goals
+
+* The Kubescape Operator automates scanning and monitoring of Kubernetes clusters
+  * provides up-to-date security posture data
+  * creates proposals on how to improve the security posture of the workload and the cluster
+  * provides security observability data
+  * alerts on potential breaches to workloads in the cluster
+* Kubescape CLI scans pre-deployment and post-deployment configurations and images and provides actionable results
+
+#### Security Goals
+
+* All access to the data produced by Kubescape Operator should be only available to those subjects who have the proper RBAC authorizations
+* Kubescape should only share information to מon-Kubernetes endpoints in cases when the user explicitly defined it
+* Kubescape should only have access to objects it needs to complete its job
+* Triggers should only come from authenticated and authorized sources
+
+### Non-goals
+
+* Kubescape is not meant to enforce access control beyond what the Kubernetes API already provides
+
+## Self-assessment use
+
+This self-assessment is not intended to provide a security audit of Kubescape or function as an independent assessment or attestation of Kubescape's security health.
+
+This document provides Kubescape users with an initial understanding of Kubescape's security, where to find existing security documentation, Kubescape's plans for security, and a general overview of Kubescape's security practices, both for the development of Kubescape and its operational security.
+
+This document provides the CNCF TAG-Security with an initial understanding of Kubescape to assist in a joint assessment necessary for projects under incubation. Taken together, this document and the joint assessment serve as a cornerstone for if and when Kubescape is preparing for a security audit.
+
+## Security functions and features
+
+In general, Kubescape is a security project; therefore, we do not list all the features here, only the "non-functional" security features.
+
+### Critical
+
+#### Kubernetes API server authentication
+
+Since Kubescape is mainly driven and uses an API server to store objects, it is critical to validate the Kubernetes API server properly.
+This is true for regular API server clients and even more crucial for the *Storage* component, which needs mTLS to function properly to prevent unwanted access.
+
+#### Denial of events
+
+Runtime detection relies heavily on eBPF and container runtime events. Allowing such inputs is critical to maintaining reliable detection capabilities.
+
+#### Masking sensitive results
+
+Since results can contain sensitive information (Kubernetes secrets, environment variables with keys) Kubescape masks these fields from the result objects
+
+### Security Relevant
```

Comment:
> A listing of security relevant components of the project with brief description. These are considered important to enhance the overall security of the project, such as deployment configurations, settings, etc. These should also be included in threat modeling.

On one hand, nothing is security relevant, as it's never in the critical path?
On the other hand, the entire project is security relevant?

I would remove the two sections below, I think they don't add value.

Reply:
I was advised to keep it like this for now to meet other formatting, but we should work on this later